Makefile

```diff
@@ -3,14 +3,34 @@ clean:
 	rm -f dist/*.whl
 	rm -rf build
 
-build:
+# PyPI archive preparation
+build_pypi:
 	python setup.py sdist
+	python -m cython -a streaming_form_data/_parser.pyx -o annotation.html
 
-upload: build
+upload: build_pypi
 	twine upload dist/*
 
-test:
+# build the module and install to the system
+install:
+	python -m pip install . --ignore-installed
```

Comment:
Or just `pip install .`.

Reply:
Hmm. 
"--ignore-installed" suppressed me warnings about some problems in uninstall. I do not remember which. Something like "missing files to uninstall"

But right now I cannot reproduce the problem.

Reply:
I found.

There is some strange difference between 'pip' call and 'python -m pip':

(for your information: in my OS python and pip are located in different folders. So when python is on the PATH, pip may be missing. This is why I prefer to call pip through 'python -m pip')

```
c:\work\github\streaming-form-data>pip install .
Processing c:\work\github\streaming-form-data
Installing collected packages: streaming-form-data
  Found existing installation: streaming-form-data 0.4.5
    Uninstalling streaming-form-data-0.4.5:
      Successfully uninstalled streaming-form-data-0.4.5
  Running setup.py install for streaming-form-data ... done
Successfully installed streaming-form-data-0.4.5

c:\work\github\streaming-form-data>python -m pip install .
Processing c:\work\github\streaming-form-data
Installing collected packages: streaming-form-data
  Found existing installation: streaming-form-data 0.4.5
    Can't uninstall 'streaming-form-data'. No files were found to uninstall.
  Running setup.py install for streaming-form-data ... done
Successfully installed streaming-form-data-0.4.5
```

**Can't uninstall 'streaming-form-data'. No files were found to uninstall.**

This happens constantly. :(